lib/Dialect/Torch/Utils/Utils.cpp

```diff
@@ -617,3 +617,19 @@ Type Torch::getDefaultAccType(PatternRewriter &rewriter, Type inputType) {
     return rewriter.getI64Type();
   return inputType;
 }
+
+SmallVector<Value> Torch::getValueVector(OpBuilder &builder, Location loc,
+                                         ArrayRef<int64_t> intVector) {
+  SmallVector<Value> ret;
+  llvm::transform(intVector, std::back_inserter(ret), [&](int64_t n) {
+    return builder.create<ConstantIntOp>(loc, builder.getI64IntegerAttr(n));
+  });
+  return ret;
+}
+
+PrimListConstructOp Torch::getList(OpBuilder &builder, Location loc,
+                                   ArrayRef<int64_t> intVector) {
+  SmallVector<Value> vals = getValueVector(builder, loc, intVector);
+  return builder.create<Torch::PrimListConstructOp>(
+      loc, Torch::ListType::get(builder.getType<Torch::IntType>()), vals);
+}
```

Comment:
This is not required. There already exists a utility which does the same, here: https://github.com/llvm/torch-mlir/blob/7e0e23c66820d1db548103acbdf1337f701dc5a3/lib/Conversion/TorchOnnxToTorch/Utils.cpp#L17-L29

Reply:
Thanks!, Now I know that exist, but I changed the util to accept an ArrayRef which is more useful here.
Also, why does it live under OnnxToTorch util ? This looks to be useful util be inside the dialect (not just the conversion)

Reply:
I think the original author might not have think about it in that way, maybe we can move it there later when required.